src/Microsoft.AspNet.Mvc.Razor/IRazorPage.cs

```diff
@@ -67,5 +68,7 @@ public interface IRazorPage
         /// <exception cref="InvalidOperationException">if one or more sections were not rendered or if no sections were
         /// defined and the body was not rendered.</exception>
         void EnsureRenderedBodyOrSections();
+
+        IHtmlContentBuilder Output { get; set; }
```

Comment:
The delegate used to be invoked from `IRazorPage.RenderBody()`. We simply return the `BodyContent` from it since there's no async work involved.